python/GangaLHCb/__init__.py

```diff
@@ -8,7 +9,7 @@
 import tempfile
 import Ganga.Utility.logging
 import Ganga.Utility.Config
-from optparse import OptionParser, OptionValueError
+from optparse import OptionParser, OptionValueError, BadOptionError
```

Comment:
I could be wrong, but I don't think `BadOptionError` is used and so can be removed here.